tests/operator/operator.go

```diff
@@ -3058,11 +3058,13 @@ spec:
 	})
 
 	Context("[Serial] Deployment of common-instancetypes", Serial, func() {
+		var appComponent string
```

Comment:
nit: Could we rather always call `apply.GetAppComponent(util2.GetCurrentKv(virtClient))` to prevent mistakes.